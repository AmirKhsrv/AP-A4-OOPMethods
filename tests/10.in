show_sorted_shelf 41 want_to_read Nonfiction
best_book
show_sorted_shelf 107 want_to_read Nonfiction
recommend_first_approach 24
best_reviewer
credit 51
recommend_first_approach 103
