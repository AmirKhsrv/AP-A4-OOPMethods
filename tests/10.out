id: 232
Title: Crocodile on the Sandbank
Genre: Mystery
Author: Elizabeth Peters
***
id: 230
Title: Fables Vol. 1: Legends in Exile
Genre: Sequential Art
Author: Bill Willingham
***
id: 367
Title: Homeland
Genre: Fantasy
Author: R.A. Salvatore
***
id: 262
Title: Kamisama Kiss Vol. 1
Genre: Sequential Art
Author: Julietta Suzuki
***
id: 403
Title: Magic Triumphs
Genre: Fantasy
Author: Ilona Andrews
***
id: 204
Title: Mockingjay
Genre: Young Adult
Author: Suzanne Collins
***
id: 310
Title: Nobody But Us
Genre: Young Adult
Author: Kristin Halbrook
***
id: 229
Title: Tales of a Fourth Grade Nothing
Genre: Childrens
Author: Judy Blume
***
id: 5
Title: The Secret Wisdom of the Earth
Genre: Fiction
Author: Christopher Scotton
***
id: 133
Title: The Strange and Beautiful Sorrows of Ava Lavender
Genre: Young Adult
Author: Leslye Walton
***
id: 378
Title: The Tycoon's Revenge
Genre: Romance
Author: Melody Anne
***
id: 46
Title: Wicked as They Come
Genre: Science Fiction
Author: Delilah S. Dawson
***
id: 166
Title: You Can Heal Your Life
Genre: Self Help
Author: Louise L. Hay
***
id: 23
Title: You're Welcome Universe
Genre: Young Adult
Author: Whitney Gardner
***
id: 405
Title: Ellen Foster
Genre: Fiction
Author: Kaye Gibbons
Average Rating: 3.42
id: 371
Title: All The Wrong Reasons
Genre: Romance
Author: J.L. Paul
***
id: 141
Title: Cassandra at the Wedding
Genre: Fiction
Author: Dorothy Baker
***
id: 15
Title: Into the Wilderness
Genre: Historical
Author: Sara Donati
***
id: 241
Title: Kuttiedathi and Other Stories
Genre: Short Stories
Author: M.T. Vasudevan Nair
***
id: 266
Title: Madhur Jaffrey's World Vegetarian: More Than 650 Meatless Recipes from Around the World
Genre: Food and Drink
Author: Madhur Jaffrey
***
id: 237
Title: North and South
Genre: Classics
Author: Elizabeth Gaskell
***
id: 381
Title: Peter the Great: His Life and World
Genre: History
Author: Robert K. Massie
***
id: 27
Title: Someday This Pain Will Be Useful to You
Genre: Young Adult
Author: Peter Cameron
***
id: 37
Title: Teaching My Mother How to Give Birth
Genre: Poetry
Author: Warsan Shire
***
id: 306
Title: The Madness Underneath
Genre: Young Adult
Author: Maureen Johnson
***
id: 350
Title: The Vanderbeekers of 141st Street
Genre: Childrens
Author: Karina Yan Glaser
***
id: 82
Title: Genghis Khan and the Making of the Modern World
Genre: History
Author: Jack Weatherford
Reviews:
id: 245 Rating: 1 Likes: 1103 Date: February 12 2011
id: 1978 Rating: 3 Likes: 0 Date: May 26 2009
id: 2419 Rating: 4 Likes: 0 Date: June 4 2018
id: 2503 Rating: 3 Likes: 0 Date: January 7 2019
id: 2563 Rating: 5 Likes: 0 Date: June 27 2016
id: 3171 Rating: 0 Likes: 0 Date: August 13 2015
id: 3338 Rating: 5 Likes: 0 Date: December 21 2017
id: 3556 Rating: 5 Likes: 0 Date: March 29 2018
id: 3774 Rating: 0 Likes: 0 Date: October 1 2011
id: 3883 Rating: 1 Likes: 0 Date: July 24 2019
id: 4155 Rating: 3 Likes: 3 Date: March 17 2017
id: 4772 Rating: 4 Likes: 0 Date: January 11 2018
id: 5291 Rating: 1 Likes: 0 Date: October 18 2019
id: 5481 Rating: 0 Likes: 0 Date: June 27 2019
id: 5674 Rating: 1 Likes: 0 Date: October 13 2018
id: 6473 Rating: 5 Likes: 4 Date: December 3 2014
id: 6568 Rating: 4 Likes: 3 Date: December 6 2018
id: 6572 Rating: 5 Likes: 6 Date: May 7 2015
id: 6611 Rating: 1 Likes: 2 Date: October 18 2018
id: 7436 Rating: 0 Likes: 2 Date: May 10 2017
id: 7886 Rating: 5 Likes: 0 Date: June 3 2018
id: 9102 Rating: 5 Likes: 0 Date: January 15 2014
id: 9385 Rating: 5 Likes: 0 Date: December 5 2019
id: 9645 Rating: 3 Likes: 0 Date: May 22 2019
id: 10158 Rating: 3 Likes: 0 Date: August 26 2014
id: 10628 Rating: 4 Likes: 1 Date: March 3 2019
id: 10638 Rating: 0 Likes: 0 Date: July 21 2012
id: 11073 Rating: 1 Likes: 0 Date: February 8 2018
id: 12096 Rating: 0 Likes: 1 Date: April 12 2018
id: 12761 Rating: 4 Likes: 0 Date: June 14 2019
id: 13298 Rating: 2 Likes: 0 Date: January 2 2018
id: 13434 Rating: 2 Likes: 0 Date: March 8 2017
id: 13684 Rating: 5 Likes: 0 Date: December 27 2019
id: 13932 Rating: 1 Likes: 0 Date: July 28 2014
id: 14519 Rating: 1 Likes: 0 Date: March 18 2016
id: 14653 Rating: 2 Likes: 1 Date: March 28 2013
id: 15057 Rating: 5 Likes: 0 Date: May 7 2017
id: 15171 Rating: 4 Likes: 0 Date: October 17 2011
id: 15763 Rating: 0 Likes: 1 Date: October 27 2018
id: 15815 Rating: 0 Likes: 0 Date: September 3 2017
id: 15968 Rating: 4 Likes: 2 Date: June 12 2018
id: 16595 Rating: 2 Likes: 0 Date: November 15 2018
id: 31
Name: Athena Souri
Place of Birth: Switzerland
Member Since: December 24 2010
Favorite Genres: Romance
Favorite Authors: Carl Sagan, Colleen Clayton, Greer Noble
Number of Books in Read Shelf: 8
Number of Books in Want to Read Shelf: 15
Number of Books in Currently Reading Shelf: 6
Number of Likes: 2455
0.006458
id: 236
Title: Slow Burn
Genre: Romance
Author: Sam B. Morgan
Reviews:
id: 22 Rating: 0 Likes: 38 Date: February 27 2016
id: 709 Rating: 4 Likes: 2 Date: September 27 2013
id: 816 Rating: 2 Likes: 0 Date: November 29 2019
id: 1315 Rating: 5 Likes: 0 Date: October 16 2015
id: 1626 Rating: 2 Likes: 0 Date: March 28 2019
id: 1698 Rating: 3 Likes: 0 Date: July 2 2016
id: 2240 Rating: 2 Likes: 0 Date: March 4 2013
id: 2338 Rating: 3 Likes: 0 Date: March 29 2019
id: 2651 Rating: 2 Likes: 0 Date: November 25 2018
id: 2702 Rating: 5 Likes: 0 Date: October 5 2018
id: 2861 Rating: 3 Likes: 0 Date: April 11 2014
id: 4305 Rating: 5 Likes: 20 Date: May 8 2015
id: 4625 Rating: 0 Likes: 1 Date: March 8 2011
id: 5276 Rating: 5 Likes: 0 Date: September 4 2019
id: 6661 Rating: 4 Likes: 2 Date: January 11 2019
id: 6698 Rating: 3 Likes: 2 Date: December 29 2010
id: 6754 Rating: 5 Likes: 5 Date: July 23 2014
id: 7207 Rating: 4 Likes: 13 Date: June 27 2018
id: 7241 Rating: 3 Likes: 12 Date: October 20 2011
id: 7335 Rating: 1 Likes: 1 Date: April 15 2012
id: 7395 Rating: 2 Likes: 0 Date: September 9 2019
id: 7785 Rating: 2 Likes: 0 Date: January 27 2019
id: 7998 Rating: 4 Likes: 0 Date: March 19 2016
id: 8669 Rating: 3 Likes: 0 Date: May 2 2014
id: 9719 Rating: 0 Likes: 0 Date: November 24 2019
id: 9874 Rating: 2 Likes: 0 Date: August 25 2014
id: 10004 Rating: 4 Likes: 0 Date: March 25 2019
id: 10171 Rating: 4 Likes: 2 Date: February 14 2019
id: 10644 Rating: 2 Likes: 0 Date: June 25 2019
id: 10714 Rating: 4 Likes: 20 Date: June 6 2019
id: 10841 Rating: 3 Likes: 0 Date: April 27 2017
id: 10909 Rating: 5 Likes: 0 Date: April 9 2013
id: 11757 Rating: 0 Likes: 0 Date: March 27 2018
id: 12179 Rating: 5 Likes: 0 Date: September 5 2015
id: 13019 Rating: 5 Likes: 1 Date: May 21 2012
id: 13321 Rating: 4 Likes: 0 Date: May 18 2016
id: 14914 Rating: 1 Likes: 0 Date: November 17 2018
id: 16115 Rating: 5 Likes: 0 Date: April 7 2016
